lib/ReactViews/ExplorerWindow/Tabs/MyDataTab/add-data.scss

```diff
@@ -4,11 +4,11 @@
 @include empty-module("inner");
 
 .tab-panels {
-  width: 50%;
-  max-width: 600px;
+  //width: 50%;
+  //max-width: 600px;
```

Comment:
We could try:

```css
width: fit-content;
min-width: 50%;
max-width: 600px
```

To keep the original width but allow room for custom components to expand if required (within the max limit).